examples/params_conn.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
maybe name the file `params_connectivity.json` so people are not wondering what `conn` stands for